docs/SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+# Support
```

Comment:
Did this actually copy the file? Would it be possible to symlink (`ln -s`)?

Reply:
I missed that that file was symlinked, since it shows up with the full text in PyCharm. Makes sense when I actually go back and look at the history of CONTRIBUTING now.